backend/main.py

```diff
@@ -35,8 +35,8 @@
 )  # include urls from sync.py
 
 
-app.mount(
-    "/",
-    StaticFiles(directory="../frontend/dist", html=True, check_dir=False),
-    name="static",
-)
+# app.mount(
+#     "/",
+#     StaticFiles(directory="../frontend/dist", html=True, check_dir=False),
+#     name="static",
+# )
```

Comment:
WHAT IS GOING ON HERE?
Why would you want to comment this out?

kindly take this out completely and let the file be as it was.
If you have any changes to make here, it must be properly discussed first